documentation/source/user_section/tutorials/spinalcord-segmentation/sct_propseg-example-t1.rst

```diff
@@ -0,0 +1,44 @@
+Hands-on Example: T1
+####################
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t1_image.png
```

Comment:
This image is a bit sad (small and very dark). Also, I'm not sure it really brings much information. I would personally remove it, or make it bigger, lighter and center it (instead of moving it to the right).